bin/generate_contributors.go

```diff
@@ -74,6 +75,7 @@ func main() {
 		"sqs":             "",
 		"todo":            "",
 		"tsenart":         "",
+		"uwedeportivo":    "",
 		"vanesa":          "",
 	}
```

Comment:
Seems like my handle is not added: `unknwon`.

Reply:
Good catch! I just added it.